tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -8,154 +8,238 @@
-                    required=False, validators=[]),
-              Field(item="60A", name='SCHOOL_ATTENDENCE_HOP', type='string', startIndex=114, endIndex=116,
-                    required=False, validators=[]),
-              Field(item="60B", name='SCHOOL_ATTENDENCE_EA', type='string', startIndex=116, endIndex=118,
-                    required=False, validators=[]),
-              Field(item="60C", name='SCHOOL_ATTENDENCE_HOL', type='string', startIndex=118, endIndex=120,
-                    required=False, validators=[]),
-              Field(item="61A", name='PROVIDE_CC_HOP', type='string', startIndex=120, endIndex=122,
-                    required=False, validators=[]),
-              Field(item="61B", name='PROVIDE_CC_EA', type='string', startIndex=122, endIndex=124,
-                    required=False, validators=[]),
-              Field(item="61C", name='PROVIDE_CC_HOL', type='string', startIndex=124, endIndex=126,
-                    required=False, validators=[]),
-              Field(item="62", name='OTHER_WORK_ACTIVITIES', type='string', startIndex=126, endIndex=128,
-                    required=False, validators=[]),
-              Field(item="63", name='DEEMED_HOURS_FOR_OVERALL', type='string', startIndex=128, endIndex=130,
-                    required=False, validators=[]),
-              Field(item="64", name='DEEMED_HOURS_FOR_TWO_PARENT', type='string', startIndex=130, endIndex=132,
-                    required=False, validators=[]),
-              Field(item="65", name='EARNED_INCOME', type='string', startIndex=132, endIndex=136,
-                    required=False, validators=[]),
-              Field(item="66A", name='UNEARNED_INCOME_TAX_CREDIT', type='string', startIndex=136, endIndex=140,
-                    required=False, validators=[]),
-              Field(item="66B", name='UNEARNED_SOCIAL_SECURITY', type='string', startIndex=140, endIndex=144,
-                    required=False, validators=[]),
-              Field(item="66C", name='UNEARNED_SSI', type='string', startIndex=144, endIndex=148,
-                    required=False, validators=[]),
-              Field(item="66D", name='UNEARNED_WORKERS_COMP', type='string', startIndex=148, endIndex=152,
-                    required=False, validators=[]),
-              Field(item="66E", name='OTHER_UNEARNED_INCOME', type='string', startIndex=152, endIndex=156,
-                    required=False, validators=[]),
-          ],
-        )
-      ]
+t2 = SchemaManager(schemas=[
+    RowSchema(
+        model=TANF_T2,
+        preparsing_validators=[
+            validators.hasLength(156),
+        ],
+        postparsing_validators=[
+                validators.validate__FAM_AFF__SSN(),
+                validators.if_then_validator(
+                    condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                    result_field='SSN', result_function=validators.notOneOf([str(i)*9 for i in range(0, 9)]),
+                ),
```

Comment:
the following check appears to be missing here. 

![2369p1](https://github.com/raft-tech/TANF-app/assets/63075587/0b136f88-7808-431b-9ac2-7fbf7d1c8d26)